### ISSUE TITLE
Error with localized usernames

### ISSUE THREAD

**Issue by psionika** [NetSparkleUpdater/NetSparkle#112]
Hello. I found a small bug in the **RunDownloadedInstaller(string downloadFilePath)** function. If the user name contains letters of a different alphabet, the update will not be performed.

Error example:
<img width="713" alt="bug" src="https://user-images.githubusercontent.com/2479338/86548595-f1fb1980-bf77-11ea-8871-056b1b35ed10.PNG">

For correction, I rewrote it like this

```
LogWriter.PrintMessage("Generating batch in {0}", Path.GetFullPath(batchFilePath));

var consoleEncoding = Encoding.GetEncoding(CultureInfo.CurrentCulture.TextInfo.OEMCodePage);
using (StreamWriter write = new StreamWriter(batchFilePath, false, consoleEncoding))
{

**Comment by Deadpikle** [comment]
Ooof, a good old UTF-8 issue! I'll make sure to address this next time I take a look at fixing/updating things (usually once a week or so). Thank you for the report!

**Comment by Deadpikle** [comment]
This is being pushed out now in `2.0.0-preview20200712001`. Thanks! (Used a UTF-8 encoder instead as your suggested fix did not work on my system.) Again, appreciate your report and help!

**Comment by psionika** [comment]
Unfortunately, your fix does not solve the problem in my case :( I tested in two versions - pure Windows 7 and Windows 10
<img width="1004" alt="Capture" src="https://user-images.githubusercontent.com/2479338/87262311-6f80d580-c4f4-11ea-882a-253cb4b76f7e.PNG">

**Comment by Deadpikle** [comment]
Thanks for the feedback and letting me know the fix didn't work.

Is that text you're using the actual username you're using? I wonder if the issue is more to do with the space in the name than what's showing up in your console (console isn't necessarily UTF-8 friendly all the time). A quick glance at the code shows that spaces in the output may cause issues; I need to dig deeper.

Would you be willing to add a comment here with that actual text in non-screenshot form so I can copy-paste it and make sure I'm testing with the right text? I don't know how to type that myself, sorry 😅 

Thanks.

**Comment by psionika** [comment]
No. 
This string "тестовая строка" is only for test. 
Problem username is "Оператор".

**Comment by Deadpikle** [comment]
Fixed by #176.
